docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+| `scrollInputMultiplier` | `{Number}` | 50 | Multiplication factor for the gamepad input value to calculate the offset from the current scroll position (in pixels). |
+
+These options can be provided to the component in the following ways:
+
+``` javascript
+fluid.registerNamespace("gamepad.inputMapper.base");
+fluid.registerNamespace("gamepad.inputMapperUtils.content");
+
+// Either pass options inside the object as an argument while creating an instance of inputMapper.base component.
+var inputMapperBaseInstanceOne = gamepad.inputMapper.base({
+    cutoffValue: 0.15,
+    scrollInputMultiplier: 35
+});
+
+// Otherwise pass it as default options in a custom component using the inputMapper.base grade.
+fluid.defaults("my.inputMapperBase.grade", {
+    gradeNames: ["gamepad.inputMapper.base"],
+    cutoffValue: 0.10,
+    scrollInputMultiplier: 25
+});
+var inputMapperBaseInstanceTwo = my.inputMapperBase.grade();
+```
+
+You can also pass custom configuration map (in the form of model variables) to specify the action and other associated
+parameters for the gamepad inputs as an argument. For information regarding the configuration map model variables, refer
+to the [Component Options](configMaps.md#component-options) section in the [`configMaps`](configMaps.md) grade document.
+
+``` javascript
+fluid.registerNamespace("gamepad.inputMapper.base");
+fluid.registerNamespace("gamepad.inputMapperUtils.content");
+
+var inputMapperBaseInstance = gamepad.inputMapper.base({
+    cutoffValue: 0.35,
+    scrollInputMultiplier: 60,
+    model: {
+        map: {
+            buttons: {
+                "0": {
+                    currentAction: "scrollDown",
+                    speedFactor: 2.2
+                }
+            }
+        }
+    }
+});
+```
+
+## Non-navigation Invokers
+
+### `{inputMapper.base}.produceNavigation(that, change)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `change {Object}` Recipt for the change in input values.
+- Returns: Nothing.
+
+Listens for the changes in gamepad's input values stored as `inputMapper.base` component's model data and calls the
+**navigation-producing invokers** according to the configuration map. The navigation-producing invoker is triggered only
+if the component provides it. Each of these invokers is passed with the following arguments:
+
+- `inputValue`: Current value of the gamepad input.
+- `speedFactor`: Times by which the speed of given (continuous) action should be increased.
+- `invert`: Whether direction of the given action should be in the opposite order (for thumbsticks).
+- `background`: Whether a new tab or window should open in background.
+- `oldInputValue`: Previous value of the gamepad input before its state is changed.
+- `homepageURL`: URL that a new tab or window should load when opened.
+
+The invokers use only those arguments required to perform the navigation it's meant for. Apart from the `inputValue` and
```

Comment:
I see what you mean, but "the navigation it's meant for" is a bit awkward.  I'd suggest "their specific action".